src/lib/core/CHIPConfig.h

```diff
@@ -1758,6 +1758,16 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_BDX_LOG_TRANSFERS 5
 #endif // CHIP_CONFIG_MAX_BDX_LOG_TRANSFERS
 
+/**
+ *  @def CHIP_CONFIG_MAX_WIFI_NETWORK
+ *
+ *  @brief
+ *    Maximum number of wifi network
+ */
+#ifndef CHIP_CONFIG_MAX_WIFI_NETWORK
+#define CHIP_CONFIG_MAX_WIFI_NETWORK 1
```

Comment:
```suggestion
#define CHIP_CONFIG_MAX_WIFI_NETWORKS 1
```
and fix all the other places that mention it.